sbt-bridge/test/xsbt/TestDriver.scala

```diff
@@ -0,0 +1,15 @@
+package xsbt
+
+import dotty.tools.dotc._
+import core.Contexts._
+
+class TestDriver extends Driver {
```

Comment:
Do you really need to make this public? Can't in `private` inside the companion object of `ScalaCompilerForUnitTesting`?